src/gui/layout/qgslayoutitemslistview.cpp

```diff
@@ -119,6 +119,30 @@ void QgsLayoutItemsListView::setCurrentLayout( QgsLayout *layout )
   connect( selectionModel(), &QItemSelectionModel::selectionChanged, this, &QgsLayoutItemsListView::updateSelection );
 }
 
+void QgsLayoutItemsListView::keyPressEvent( QKeyEvent *event )
+{
+  if ( event->key() == Qt::Key_Space )
```

Comment:
Can you confirm that this doesn't break the ability to enter a space character in the list when renaming items?

Reply:
Good input, thanks. I double checked. It does not break the renaming.